cmd/serverless/api_key.go

```diff
@@ -77,7 +79,9 @@ func readAPIKeyFromKMS(cipherText string) (string, error) {
 	if cipherText == "" {
 		return "", nil
 	}
-	sess, err := session.NewSession(nil)
+	sess, err := session.NewSession(aws.NewConfig().WithHTTPClient(&http.Client{
+		Transport: datadogHttp.CreateHTTPTransport(),
+	}))
```

Comment:
Difficult to add tests, and `CreateHTTPTransport` is already tested so I believe it would be redundant.